cygrid/tests/test_cygrid.py

```diff
@@ -58,26 +62,132 @@ def setup(self):
             'CRVAL2': mapcenter[1],
             }
 
-    def teardown(self):
-
-        pass
-
-    def test_gridding(self):
-
         kernelsize_fwhm = self.beamsize_fwhm / 2
         kernelsize_sigma = kernelsize_fwhm / 2.35
         support_radius = 3. * kernelsize_sigma
         hpx_min_res = kernelsize_sigma / 2.
 
-        mygridder = WcsGrid(self.header)
-        mygridder.set_kernel(
+        self.kernel_args = (
             'gauss1d',
             (0.5 / kernelsize_sigma ** 2,),
             support_radius,
             hpx_min_res,
             )
 
-        mygridder.grid(self.xcoords, self.ycoords, self.signal[:, np.newaxis])
+        self.test_maps = np.load(get_pkg_data_filename(
+            'data/cygrid_test_maps.npy'
+            ))
+
+    def teardown(self):
+
+        pass
+
+    def _do_gridding2d(self, signal, header, **kwargs):
+
+        mygridder = cygrid.WcsGrid(header, **kwargs)
+        mygridder.set_kernel(*self.kernel_args)
+
+        mygridder.grid(self.xcoords, self.ycoords, signal[:, np.newaxis])
```

Comment:
Wasn't the purpose of this patch to get rid of the degenerate axis if one wants to grid maps? Sorry if I'm misunderstanding this!